### PR TITLE
Modifications to Assignment 4

### PR DESCRIPTION
**Pull request by gildedgardenia**

### CODE REVIEW

File: src/assignments/html-me-part1.rst [hunk 1]
```diff
@@ -144,4 +143,4 @@ coherent chunk of work.
 Done!
 ------
 
-Well done! Time to `dive into some CSS <html-me-part2.html>`__.
+Well done! You may notice that you haven't got a green check mark on your repository yet. Time to `dive into some CSS <html-me-part2.html>`__.
```
Comment (CarlyLanglois) [inline src/assignments/html-me-part1.rst:148]: I like this note but it may be more relevant if the students have been directed to commit and push

**Review by CarlyLanglois — APPROVED** [review]
This looks good Sally - the biggest thing is that the students will need to know that they must push their changes to their remote assignment repository. I think this should be included in the instructions - maybe in the submission section or before.

Also - what do you think about adding some language about viewing the starter code repo and exploring the relevant and irrelevant files once they have that cloned.

I don't need to review this again but if you need me to clarify, hit me up

**Comment by CarlyLanglois** [comment]
[Independent Track: Assignment 4 Autograding](https://trello.com/c/fGSpyeah/660-independent-track-assignment-4-autograding)

File: src/assignments/html-me-part1.rst (hunk 1 above)
Comment (gildedgardenia) [reply src/assignments/html-me-part1.rst:148]: Added additional language to direct students to commit and push! Thanks!

**Pull request merged**